src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoLineWrapCheck.java

```diff
@@ -45,6 +45,9 @@
  * STATIC_IMPORT</a>.
  * </li>
  * </ul>
+ * <p>
+ * To configure the check:
+ * </p>
```

Comment:
Combine

Reply:
Same as above

Reply:
https://checkstyle.org/config_whitespace.html#NoLineWrap

swap example and config and no extra `To configure the check` will be requied

Reply:
Done